dace/sdfg/sdfg.py

```diff
@@ -469,6 +496,21 @@ def arrays(self):
         """
         return self._arrays
 
+    @property
```

Comment:
A property of a Property? bound to be messy!

Reply:
Same as `SDFG::_arrays`, `SDFG::arrays`. If we consider that the array descriptor store does not follow the DaCe programming conventions, then it should be fixed ASAP.